packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/ScatterPlotViewer/README.md

```diff
@@ -67,7 +70,48 @@ The JSON file is a very, very basic data object consisting of an array of number
 }
 ```
 
-A set of chart options can be defined along with the data that define the x-axis and y-axis labels as well as the margins and padding to use. Padding is defined as the the space inside the axes lines. Defined padding will likely only be used by scatter plots using an inner tick direction similar to the current PH: TESS light curve viewer. Margin is defined as the space outside axes lines. Defined margin should be used by the outer tick direction which is the default orientation for the scatter plot axes. 
+The multiple series JSON shape is an array of objects consisting of `seriesData` and `seriesOptions` properties:
+
+``` json
+{ "data": [
+    { 
+      "seriesData": [
+        { "x": 1.46,
+          "y": 6.37,
+          "x_error": 2,
+          "y_error": 0.5
+        }, {
+          "x": 7.58,
+          "y": 9.210
+        }
+      ],
+      "seriesOptions": {
+        "color": "accent-2",
+        "label": "Filter 1"
+      }
+    }, {
+      "seriesData": [
+        { "x": 700,
+          "y": 500,
+          "x_error": 2,
+          "y_error": 0.5
+        }, {
+          "x": 701,
+          "y": 900
+        }
+      ],
+      "seriesOptions": {
+        "color": "#98b6a7",
+        "label": "Filter 2"
+      }
+    }
+  ]
+}
+```
+
+The `seriesData` property should be an array of objects where at minimum an x and y coordinate is required. An optional `x_error` and/or `y_error` number can be specified if error bars need to be displayed for that single data point. Each series supports a set of options under `seriesOptions` and at minimum a string `label` is required for each series. An optional string `color` for can defined using either a variable name from the colors available in from the [zooniverse theme object](https://github.com/zooniverse/front-end-monorepo/tree/master/packages/lib-grommet-theme) or a hex value. If a color is not provided, a color from the zooniverse theme will be chosen and applied for each series. 
+
+For both single series data and multiple series data, a set of chart options can also be supplied that define the x-axis and y-axis labels as well as the margins and padding to use. Padding is defined as the the space inside the axes lines. Defined padding will likely only be used by scatter plots using an inner tick direction similar to the current PH: TESS light curve viewer. Margin is defined as the space outside axes lines. Defined margin should be used by the outer tick direction which is the default orientation for the scatter plot axes. 
```

Comment:
```suggestion
For both single series data and multiple series data, a set of chart options can also be supplied that define the x-axis and y-axis labels as well as the margins and padding to use. Padding is defined as the space inside the axes lines. Defined padding will likely only be used by scatter plots using an inner tick direction similar to the current PH: TESS light curve viewer. Margin is defined as the space outside axes lines. Defined margin should be used by the outer tick direction which is the default orientation for the scatter plot axes. 
```

Reply:
GitHub doesn't show my changes very clearly. I changed 'the the' to 'the' in the second sentence.